interfaces/hotplug/spec.go

```diff
@@ -42,9 +42,9 @@ type RequestedSlotSpec struct {
 	// Name is how the interface wants to name the slot. When left empty,
```

Comment:
I would rename this struct  AddedSlotSpec

Reply:
Hmm, I'm not sure, "AddedSlotSpec" sounds like the slot was (unconditionally) added, while the idea is the interface creates a spec and we mediate before adding the slot for real.

Reply:
we can chat about this tomorrow, for sure Requested is confusing in its own ways

Reply:
now I looked a bit more at this and I remember I was already a bit uncomfortable with the complexity of the Specification stuff for the hotplug case, we do need to chat a bit about this